ui/templates/main.html

```diff
@@ -56,6 +56,7 @@ <h1>Φορτίο (fortio) v{{.Version}}{{if not .DoLoad}} control UI{{end}}</h1>
 Load using http: <input type="radio" name="runner" value="http" checked/> or
 grpc: <input type="radio" name="runner" value="grpc"/>
 (grpc secure transport (tls): <input type="checkbox" name="grpc-secure"/>) <br />
+CA Certificate to use when grpc secure transport (tls) is selected: <input type="text" name="cacert" size="64" value="/etc/ssl/certs/ca-certificates.crt" /> <br />
```

Comment:
I still think this is not useable/useful - on the UI now you have to set 2 things to make things work

Reply:
I am going to remove my ui changes from this PR. I will open an issue to address adding tls to the ui.